### ISSUE TITLE
Update @cloudflare/kv-asset-handler

### ISSUE THREAD

**Issue by minddust** [cloudflare/worker-sites-init#6]
template version 0.0.5 doesn't support `serveSinglePageApp`. 

current available version: 0.0.7

**Comment by ashleymichal** [comment]
using `^0.0.5` should be allowing it to adapt all the way up to `1.0.0`, however checking in `package.lock` means that we pin unintentionally (I believe).

**Comment by ashleymichal** [comment]
@minddust i think your problem can be solved by removing your local package.lock file and re-running `npm install`. in the meantime we'll discuss whether it makes sense to continue committing the package.lock file to this repo.

**Comment by minddust** [comment]
i know how to fix stuff, that's not the issue. it's to stay up-to-date or to remove the lock as you suggested ;) makes it easier for starters not to struggle with a legacy version

**Comment by ashleymichal** [comment]
understood! i always try to provide the unblock suggestion if only to benefit the next user who stumbles upon this issue 😃

**Comment by kristianfreeman** [comment]
i think we should bump the package version and keep package-lock.json! afaik that's the right way to do this ~cc @ashleygwilliams as she will probably know the definitive answer~ talked to IRL, this is her suggested approach too 👍

**Comment by abury** [comment]
I've been struggling to get this to work. After bumping the  version number to 0.0.7 manually  `wrangler publish/dev` explodes saying it can't find @cloudflare/kv-asset-handler). I'll open a separate ticket for this
